iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/posix_access_rights.hpp

```diff
@@ -68,8 +68,8 @@ class PosixUser
     using groupVector_t = cxx::vector<PosixGroup, MaxNumberOfGroups>;
     using string_t = cxx::string<100>;
 
-    PosixUser(uid_t f_id);
-    PosixUser(const string_t& f_name);
+    explicit PosixUser(const uid_t f_id);
+    explicit PosixUser(const string_t& f_name);
```

Comment:
This might count as public API change and should be mentioned in the changelog. @dkroenke what do you think?

Reply:
Added an entry in changelog for that

Reply:
@FerdinandSpitzschnueffler You are right, I reworked the Changelog accordingly.